webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -9,14 +9,14 @@ import { removeLastSlashFromPath } from '../../common/helpers';
 export const getCurrentPath = () =>
   removeLastSlashFromPath(window.location.pathname);
 
-export const getActive = (data, path) => {
-  let activeItem = '';
-  data.forEach(item => {
-    item.children.forEach(child => {
-      if (child.url === path) activeItem = item.name;
-    });
-  });
-  return { title: activeItem };
+export const getActive = (items, path) => {
+  for (let i = items.length; i--; ) {
+    for (let j = items[i].children.length; j--; ) {
```

Comment:
Will `for...of` syntax will make it more readable?

Reply:
well yes, but i'm using reverse loop here, because the first child's href is `'/' which is dashboard. it meets the condition everytime so the active menu item will always be dashboard, thats why i'm doing reverse ( reverse is also faster )

Reply:
Does it mean that if the current url doesn't match any item it will end up matching the dashboard?
Shouldn't we use `exact` as react-router use https://reacttraining.com/react-router/web/api/Route/exact-bool?

Reply:
After `exact` implemented, can you migrate to `for...of`?